src/components/sections/features.js

```diff
@@ -17,38 +15,31 @@ const FeatureCard = styled(Box)(({ theme }) => ({
   alignItems: "center",
 }));
 
+export const GridItem = ({
+  image,
+  title,
+  subheading,
+  altText,
+  columnWidth = 3,
+}) => (
+  <Grid item md={columnWidth === 2 ? 6 : 4} sm={6} xs={12}>
+    <FeatureCard>
+      <Box sx={{ minHeight: 100 }}>
+        <img
+          src={image}
+          style={{ display: "block", height: 75 }}
+          loading="lazy"
+          alt={altText}
+        />
+      </Box>
+      <Box>
+        <Typography mb="1rem" sx={{ fontWeight: "bold" }}>
```

Comment:
you can remove the `<Typography/>` components here and replace it with `{children}` to bring in the mdx text (see `/content/home/helxFeatures.mdx` and `/content/eduhelx/eduHelxFeatures.mdx`)